android/canonical/app/src/androidTest/java/com/google/samples/quickstart/canonical/LoginFragmentTest.kt

```diff
@@ -0,0 +1,93 @@
+package com.google.samples.quickstart.canonical
+
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.matcher.RootMatchers
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.rule.ActivityTestRule
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiSelector
+import org.hamcrest.core.IsNot
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class LoginFragmentTest {
+    @get:Rule
+    var activityRule: ActivityTestRule<MainActivity>
+            = ActivityTestRule(MainActivity::class.java)
+
+    lateinit var device : UiDevice
+
+    @Before
+    fun setup() {
+        device = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
+        Espresso.onView(ViewMatchers.withId(R.id.sign_in_button))
+            .perform(ViewActions.click())
+        val googleSignInDialog = device.findObject(UiSelector().text("sjtuly1996@gmail.com"))
```

Comment:
Same comment here, avoid using a real account you personally own. If there is a mechanism to setup a fake/test account, use that.